site/docs/guide/deployment-types.md

```diff
@@ -359,7 +379,7 @@ However, while this behavior has some valid use cases, such a solution usually c
 Remember that once you respond to a webhook request, Telegram will send the next update for that chat.
 However, as the old update is still being processed, two updates which were previously processed sequentially, are suddenly processed in parallel.
 This can lead to race conditions.
-For example, the session plugin will inevitably break due to [WAR](https://en.wikipedia.org/wiki/Hazard_(computer_architecture)#Write_after_read_(WAR)) hazards.
+For example, the session plugin will inevitably break due to [WAR](<https://en.wikipedia.org/wiki/Hazard_(computer_architecture)#Write_after_read_(WAR)>) hazards.
```

Comment:
What are those for

Reply:
I mean the lts and the gts

Reply:
My markdown auto-formatter apparently had a bad update that does not handle links properly anymore. I reverted.